src/grid.h

```diff
@@ -118,8 +122,8 @@ class OneDimGrid
     dvec cfp; //!< Coefficient for y[j+1] in first centered difference
     dvec dlj; //!< Average of left and right grid spacing
     dvec hh; //!< Grid spacing between x[j] and x[j+1]
-    dvec rphalf; //!< "radius" at x[j+1/2]. Equal to 1 for planar geometries.
-    dvec r; //!< "radius" at x[j]. Equal to 1 for planar geometries.
+    dvec rx_half; //!< "radius" at x[j+1/2]. Equal to 1 for planar geometries.
+    dvec rx; //!< "radius" at x[j]. Equal to 1 for planar geometries.
```

Comment:
What's the logic for renaming these variables? I realize these names aren't necessarily the best, but I don't  understand what `rx` means either. Does this fit somehow with the notation you're using for the unified formulation?

Reply:
With the unified formulation, either x or z could be the radial coordinate depending on whether its a disc or cylindrical flame, so I wanted something that reflected that it was 'r in the x coordinate direction.' I know the new name still isn't ideal.